### ISSUE TITLE
Provide solution for storing "in process" information

### ISSUE THREAD

**Issue by dchandekstark** [duke-libraries/ddr-models#456]
We need to store and index information such as: the object has been queued for permanent id assignment.

**Comment by dchandekstark** [comment]
Looks like the resque API may provide enough info that we can avoid the extra complexity.

```
> Resque.size('file_characterization')
 => 166 
> Resque.peek('file_characterization', 0, 1000).map { |job| job["args"].first }
 => ["duke:296240", "duke:296239", "duke:296238", "duke:296237", "duke:296236", …
```

**Comment by dchandekstark** [comment]
Closed via #464
